.github/workflows/ci.yml

```diff
@@ -77,6 +83,7 @@ jobs:
         run: cargo check --workspace --all-targets --all-features
 
       - name: Cargo test
-        run: cargo test --workspace --all-features
+        run: cargo test -p xmr-btc test_tor_control_port --all-features
```

Comment:
Why this change please?

Reply:
To get the tests running :D Obviously this should not have been in this PR.